### PR TITLE
(example) add new example

### PR DESCRIPTION
**Pull request by dsboo**
add hello server example with custom config.

### CODE REVIEW

**Comment by dsboo** [comment]
#311

File: examples/hello_custom_config.rs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use iron::prelude::*;
+use iron::status;
+use iron::Protocol;
+use iron::Timeouts;
+
+fn main() {
+    Iron::new(|_: &mut Request| {
+        Ok(Response::with((status::Ok, "Hello world!")))
+    })
+    .listen_with("localhost:3000",
```
Comment (untitaker) [inline examples/hello_custom_config.rs]: Please put this on a newline, that's too much indentation.

**Comment by dsboo** [comment]
@untitaker thanks for your review, i change that, please review again.

**Comment by untitaker** [comment]
Thanks!

**Pull request merged**